configs/next/packages/gcc/sources

```diff
@@ -55,7 +55,7 @@ atsrc_get_patches ()
 	# Avoid a misconfiguration of tuned libraries as seen in issue #204.
 	at_get_patch \
 		'https://raw.githubusercontent.com/powertechpreview/powertechpreview/master/GCC%20PowerPC%20Backport/9/0001-libssp-Ignore-vsnprintf-test-when-ssp_have_usable_vs.patch' \
```

Comment:
@er-1 Shouldn't the new patch be pointing to commit ID 9b646544919d52be95d0190d13187d6ce872ae4d which has your new version of the patch?

Reply:
I agree, it'd be a better idea to point a commit ID instead of the head of a branch. Currently it's the same thing but if `master` changes it'll lead to an issue.